docs_input/notebooks/exercises/solutions/example2_assignment1.cu

```diff
@@ -78,10 +80,10 @@ int main() {
   /****************************************************************************************************
    * Add 5.0 to all elements of A and store the results back in A
    ****************************************************************************************************/
-  (A = A + 5.0).run();
+  (A = A + 5.0).run(exec);
```

Comment:
Thanks for adding this, but for these notebook examples I would leave out the executor and sync. These are intended for brand new users to learn the syntax, so I don't know if we should introduce syncs when they're learning the syntax. This could be added to a new notebook section, though.

Reply:
On second thought, just leave this in